x/mint/types/minter_test.go

```diff
@@ -64,14 +19,14 @@ func TestBlockProvision(t *testing.T) {
 		annualProvisions int64
 		expProvisions    int64
 	}{
-		{secondsPerYear / 5, 1},
-		{secondsPerYear/5 + 1, 1},
-		{(secondsPerYear / 5) * 2, 2},
-		{(secondsPerYear / 5) / 2, 0},
+		{secondsPerYear / 5, 121375},
+		{secondsPerYear/5 + 1, 121375},
+		{(secondsPerYear / 5) * 2, 242750},
+		{(secondsPerYear / 5) / 2, 60687},
```

Comment:
How were these expected provisions obtained

Reply:
It's a wrong test - updated it